### PR TITLE
[amplipi] Add new power channel to zones and groups

### PR DESCRIPTION
**Pull request by kaikreuzer**

### CODE REVIEW

**Review by lsiepel — CHANGES_REQUESTED** [review]
Allmost perfect, the thingTypeVersion needs to be set for both things.

```
<properties>
    <property name="thingTypeVersion">1</property>
  </properties>
```

**Comment by kaikreuzer** [comment]
Thanks, good to know! It's the first time I am using this feature.

File: bundles/org.openhab.binding.amplipi/src/main/java/org/openhab/binding/amplipi/internal/AmpliPiGroupHandler.java [hunk 1]
```diff
@@ -159,10 +164,12 @@ public void receive(Status status) {
     private void updateGroupState(Group state) {
         this.groupState = state;
 
+        Boolean power = !groupState.getMute();
```
Comment (lsiepel) [inline bundles/org.openhab.binding.amplipi/src/main/java/org/openhab/binding/amplipi/internal/AmpliPiGroupHandler.java:167]: Just a question to asses if this is intended. As it is now, power is allways the inverted value of mute. The readme shows  "Whether the zone/group is active or off", is it right that it is not a switch to power on/off but just to group (un)mute?

Comment (kaikreuzer) [reply bundles/org.openhab.binding.amplipi/src/main/java/org/openhab/binding/amplipi/internal/AmpliPiGroupHandler.java:167]: Yes, technically, power is nothing else than muting a zone/group.
I nonetheless wanted to add this channel as a solution to the problem described in https://github.com/openhab/openhab-core/pull/3990.

**Review by lsiepel — APPROVED** [review]
LGTM

**Comment by J-N-K** [comment]
So `power` is just the inverse of `mute`? It feels a bit weird to have to channels for the same physical property.

**Comment by kaikreuzer** [comment]
Yes, it feels a bit weird, but effectively, this is what happens. And being able to turn a zone on/off is an important feature.

**Review by J-N-K — APPROVED** [review]
Thanks

**Pull request merged**